src/plugins/discover/public/application/components/doc_viewer/doc_viewer.scss

```diff
@@ -1,5 +1,5 @@
 .kbnDocViewerTable {
-  @include euiBreakpoint('xs', 's') {
```

Comment:
also included in this PR: setting table-layout to `fixed` also for the `m` breakpoint to prevent this rendering using EUIDataGrid:

<img width="787" alt="Bildschirmfoto 2021-03-23 um 09 53 00" src="https://user-images.githubusercontent.com/463851/112122568-b4e7bd80-8bc0-11eb-93dd-2851b95d8a41.png">